style.css

```diff
@@ -1 +1,401 @@
 /*  Add your styling here */
+* {
+    font-family: 'Texturina', serif;
+}
+
+body {
+    width: 100%;
+}
+
+header {
+    background-color: #d9e4f5;
+    background-image: linear-gradient(315deg, #d9e4f5 0%, #f5e3e6 74%);
```

Comment:
Beautiful use of gradient 🌟

Reply:
thank you for your positive comments and your constructive feedback :-)